source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnRenderer.mm

```diff
@@ -6,48 +6,46 @@
 //
 
 #import "ACRColumnRenderer.h"
+#import "ACRColumnView.h"
 #import "ACRRegistration.h"
 #import "Column.h"
 #import "SharedAdaptiveCard.h"
 
 @implementation ACRColumnRenderer
 
-+ (ACRColumnRenderer* ) getInstance
```

Comment:
looks like changing styles seems to be more than we want to be. we can leave as it was before. in that way, we don't have to concern about updating all files.

---
In reply to: [128599993](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128599993) [](ancestors = 128599993,128594827,128563718,128559390)